app/services/content_changes.rb

```diff
@@ -1,27 +1,20 @@
-# Changes since the user's last visit
+# Content changes since the user's last visit, powered by Ahoy::Visit
 #
-# ContentChanges (powered by visits) might be more appropriate?
-class VisitChanges
+class ContentChanges
   extend Dry::Initializer
 
   option :user, required: true
 
   # @return [Boolean]
   def new_modules?
-    # for consistency change me too
+    return false if previous_visit.nil?
+
     previous_visit && new_modules.any?
```

Comment:
It'd just be `new_modules.any?` because the first bit is essentially the guard clause again

Reply:
[5855be9](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/924/commits/5855be93c98ffcb7beb854fe432292b0a5ef19b6)